### ISSUE TITLE
Signed in to unknown Google account automatically

### ISSUE THREAD

**Issue by ae5960e8-a6fc-491f-b252-898ecf59af95** [zerodytrash/Simple-YouTube-Age-Restriction-Bypass#184]
### What happened?

I just clicked onto an age-restricted video and got logged in as the account in the screenshot below. This is not my account, is it yours?

### Platform

Userscript

### Version

Latest

### What browser are you on?

Firefox

### Browser version

1111.0.1

### Screenshots or Videos

![image](https://user-images.githubusercontent.com/76442625/228669056-271bc551-4506-4d8e-83a4-bb0b43ee1319.png)

### About you

- [X] I am a developer and/or have knowledge in programming

**Comment by zerodytrash** [comment]
Oh yes, this my account!

**Comment by drunkwinter** [comment]
@ae5960e8-a6fc-491f-b252-898ecf59af95 Do you know how we might trigger this popup? It would be helpful if we could repro this.

**Comment by ae5960e8-a6fc-491f-b252-898ecf59af95** [comment]
No, it is random. Though, now this has happened, it keeps happening, and the account has not gone away or signed out after refreshing and clearing cache. That's not your real name, is it?

**Comment by drunkwinter** [comment]
Interesting. We might have to pick and send the necessary data only to avoid this.

> No, it is random. Though, now this has happened, it keeps happening, and the account has not gone away or signed out after refreshing and clearing cache. That's not your real name, is it?

No, it's his throwaway account apparently, only used for the proxy.

**Comment by D3SOX** [comment]
@drunkwinter FYI you can still see the image with the email in the edit history. If you want it to be removed you need to delete that
![Screenshot_20230330-121357_Bromite_1](https://user-images.githubusercontent.com/24937357/228804944-cecf2818-934b-4f8c-99bc-a5f7d861a4c8.png)

**Comment by drunkwinter** [comment]
@D3SOX I know i just don't want it to be easily accessible. Now I have to remove both comments if you don’t mind ;)
